basic/04-web3js-truffle/README.md

```diff
@@ -1,87 +1,103 @@
 ## Truffle 介绍
-Truffle是基于Solidity语言的一套开发框架，它简化了去中心化应用（Dapp）的构建和管理流程。本身是采用Javascript编写，支持智能合约的编译、部署和测试。
```

Comment:
just move lines and fix little, but it shows you have added +498 and delete -495, actually maybe just 3 lines changed...
you can run git fetch and merge first next time

Reply:
Maybe formatting with prettier will cause this problem.